src/dart-2.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: default
-permalink: /dart-2.0
-title: Dart 2.0 Updates
-description: How Dart 2.0 is different from Dart 1.x, and how you can prepare.
+permalink: /dart-2
+title: Dart 2 Updates
+description: How Dart 2 is different from Dart 1.x, and how you can prepare.
 ---
 
-This page has information about changes that are coming in Dart 2.0,
+This page has information about changes that are coming in Dart 2,
 and how you can migrate your code from Dart 1.x.
 
 <aside class="alert alert-warning" markdown="1">
-**Don't rely on 2.0 until it's stable.**
-Dart 2.0 pre-releases will have a series of incompatible changes.
-Although we appreciate your willingness to prepare for and test Dart 2.0,
+**Don't rely on Dart 2 until it's stable.**
+Dart 2 pre-releases will have a series of incompatible changes.
+Although we appreciate your willingness to prepare for and test Dart 2,
 please don't bet your livelihood on a pre-release.
 For information on potential changes, see
 [Dart Language & Library Newsletters.](https://github.com/dart-lang/sdk/blob/master/docs/newsletter/README.md#dart-language-and-library-newsletters)
```

Comment:
Rather than link only to the news letters, this should probably now link back to /guides/language/spec#dart-2-changes.